src/themes/default/components/Input/_input.scss

```diff
@@ -11,7 +11,7 @@
         outline: 0 none;
         background-color: transparent;
         color: $grey-dark;
-        font-size: rem(20);
+        font-size: 16px;
```

Comment:
please use the rem mixin. `rem(20)`

Reply:
`rem(20)` was failing when I tested on my local machine. Not sure if the function hasn't been imported or if it doesn't exist etc